### PR TITLE
[email] align to skill template

### PR DESCRIPTION
**Pull request by bobokids**
### Purpose
*What is the context of this pull request? Why is it being done?*
email skill align to skill template

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

**Comment by xhr0804** [comment]
>                 await sc.Context.SendActivityAsync(reply);

“cancellationToken” is missing

---
Refers to: skills/csharp/emailskill/Dialogs/ForwardEmailDialog.cs:108 in 1ec22fe. [](commit_id = 1ec22fe8bfa075382d51fb120f60475ba211b679, deletion_comment = False)

**Comment by xhr0804** [comment]
>                     return await sc.ReplaceDialogAsync(Actions.Display, skillOptions);

missed “cancellationToken”

---
Refers to: skills/csharp/emailskill/Dialogs/ShowEmailDialog.cs:396 in 1ec22fe. [](commit_id = 1ec22fe8bfa075382d51fb120f60475ba211b679, deletion_comment = False)

File: skills/csharp/emailskill/Services/BotServices.cs [hunk 1]
```diff
@@ -27,24 +27,31 @@ public BotServices(BotSettings settings, IBotTelemetryClient client)
...
                 if (config.LanguageModels != null)
                 {
                     foreach (var model in config.LanguageModels)
                     {
                         var luisApp = new LuisApplication(model.AppId, model.SubscriptionKey, model.GetEndpoint());
-                        set.LuisServices.Add(model.Id, new LuisRecognizer(luisApp, luisOptions));
+                        luisOptions = new LuisRecognizerOptionsV3(luisApp)
+                        {
+                            TelemetryClient = telemetryClient,
+                            LogPersonalInformation = true,
```
Comment (xhr0804) [inline skills/csharp/emailskill/Services/BotServices.cs]: should add 
`PredictionOptions = new Microsoft.Bot.Builder.AI.LuisV3.LuisPredictionOptions() { IncludeInstanceData = true }`
for _instance is closed by default in luisV3 but the email use it.

Comment (bobokids) [reply skills/csharp/emailskill/Services/BotServices.cs]: thanks for pointing out the issue. fixed.

**Pull request merged**